sass/theme/wrbb-styles/searchform.scss

```diff
@@ -0,0 +1,89 @@
+form.searchform{
+  position:absolute;
+  left:94%;
+  top:45%;
+  transform: translate(-70%,-50%);
+  transition: all 1s;
+  width: 50px;
+  height: 50px;
+  background: #F5F4F4;
+  box-sizing: border-box;
+  border-radius: 25px;
+  border: 4px solid #F5F4F4;
+  padding: 5px;
+}
+
+#search-input {
+  border: 0px;
+  font-size: 1em;
+  border-radius: 20px;
+  display: none;
+}
+
+.fa{
+  box-sizing: border-box;
+  padding: 10px;
+  width: 42.5px;
+  height: 42.5px;
+  position: absolute;
+  top: 0;
+  right: 0;
+  border-radius: 50%;
+  color: #07051a;
+  text-align: center;
+  font-size: 1.2em;
+  transition: all 1s;
+}
+
+form.searchform:hover{
+  width: 200px;
+  cursor: pointer;
+}
+
+form.searchform:hover #search-input{
+  display: inline-block;
+  height: 100%;
+  background-color: #F5F4F4;
+}
+
+/* Mobile styles */
+@media only screen and (max-width: 767px) {
+  form.searchform{
```

Comment:
Styles don't need to be repeated within the media queries, could you delete the duplicates?